Mathlib/Algebra/Ring/Parity.lean

```diff
@@ -88,7 +88,6 @@ lemma Even.pow_of_ne_zero (ha : Even a) : ∀ {n : ℕ}, n ≠ 0 → Even (a ^ n
 /-- An element `a` of a semiring is odd if there exists `k` such `a = 2*k + 1`. -/
 def Odd (a : α) : Prop := ∃ k, a = 2 * k + 1
 
-set_option linter.deprecated false in
 lemma odd_iff_exists_bit1 : Odd a ↔ ∃ b, a = 2 * b + 1 := exists_congr fun b ↦ by rw [two_mul]
```

Comment:
Should this be renamed? Also, the proof now can be just `Iff.rfl`, so this can be `odd_def`

Reply:
Follow-up PR please.